test/library/encryption/helpers.go

```diff
@@ -47,8 +47,19 @@ type EncryptionKeyMeta struct {
 	Mode     string
 }
 
+func SetUp(t *testing.T, verboseTearDown ...bool) testing.TB {
```

Comment:
am not a fan of these signatures.  Compare https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis. "variadic configuration" is more future proof.